README.md

```diff
@@ -118,7 +118,7 @@ The setup instructions assume that you're using [Git Bash](https://git-scm.com/)
 4. Assign the original repository to a remote called "upstream".
 
   ```sh
-  git remote add upstream https://github.com/quran/quran.com-frontend-v2
+  git remote add upstream https://github.com/quran/quran.com-frontend-v2.git
```

Comment:
if we're going with SSH,  then probably should add the correct SSH URL. That is `git@github.com:quran/quran.com-frontend-v2.git`

Reply:
I've added, SSH URL, also found a typo in 3.